content/api/commands/session.md

```diff
@@ -0,0 +1,273 @@
+---
+title: session
+---
+
+Use the `session` command to preserve the browser's state between tests. Once the steps to establish a session are performed, the session can be restored at a later time without executing those steps again.
```

Comment:
```suggestion
Preserve and restore sessions, which consist of browser [cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies), [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), and [`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) between tests.
```

Reply:
My previous suggestion (above) was just a pass to be more direct and concise. We still need to mention something along the lines of:

> The `session` command provides a way to define and reuse a "session" with an application under test to:
> - list out 3 top-level use cases
> - and more...

We also need an info alert box that reminds people that Cypress clears cookies and localStorage between tests. It should also have a one-sentence explainer on why we do that.

Reply:
Thanks. It should look better now.